src/patients/care-goals/CareGoalForm.tsx

```diff
@@ -87,50 +85,68 @@ const CareGoalForm = (props: Props) => {
       </Row>
       <Row>
         <Column sm={12}>
-          <SelectWithLabelFormGroup
-            name="priority"
-            label={t('patient.careGoal.priority.label')}
-            isRequired
-            options={priorityOptions}
-            defaultSelected={priorityOptions.filter(({ value }) => value === priority)}
-            isEditable={!disabled}
-            isInvalid={!!careGoalError?.priority}
-            onChange={onPriorityChange}
-          />
+          <div className="form-group" data-testid="prioritySelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.priority.label')}
+              htmlFor={t('patient.careGoal.priority.label')}
+              isRequired
+            />
+            <Select
+              id="prioritySelect"
+              options={priorityOptions}
+              onChange={onPriorityChange}
+              defaultSelected={priorityOptions.filter(({ value }) => value === priority)}
+              isInvalid={!!careGoalError?.priority}
+              disabled={disabled}
+            />
+          </div>
         </Column>
       </Row>
       <Row>
         <Column sm={6}>
-          <SelectWithLabelFormGroup
-            name="status"
-            label={t('patient.careGoal.status')}
-            isRequired
-            options={statusOptions}
-            defaultSelected={statusOptions.filter(({ value }) => value === status)}
-            isEditable={!disabled}
-            isInvalid={!!careGoalError?.status}
-            onChange={(values) => {
-              onFieldChange('status', values[0])
-              setStatus(values[0] as CareGoalStatus)
-            }}
-          />
+          <div className="form-group" data-testid="statusSelect">
+            <Label
+              title="This is a required input"
+              text={t('patient.careGoal.status')}
+              htmlFor={t('patient.careGoal.status')}
```

Comment:
What is this htmlFor attribute doing? I checked out https://react-bootstrap.netlify.app/components/forms/#form-label-props, and it seems it should correspond to some form id. Maybe `statusSelect`?